### PR TITLE
Math Magician : Full Website

### PR DESCRIPTION
**Pull request by sja-thedude**
Create a website consisting of 3 pages: Home, Calculator, and Quote.
The general layout will match the wireframes provided by the Microverse curriculum project requirements.
Added styles to improve the look and feel.
Used importing CSS and CSS modules.
Used React-Router to make the routes work.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @sja-thedude 

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js [hunk 1]
```diff
@@ -1,12 +1,29 @@
 /* eslint-disable linebreak-style */
```
Comment (mikenath223) [inline src/App.js]: - [x] Kindly remove this eslint disable line and fix the issue in the code.

File: src/components/Footer.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+/* eslint-disable linebreak-style */
```
Comment (mikenath223) [inline src/components/Footer.js]: - [x] Kindly remove this eslint disable line and fix the issue in the code.

File: src/components/Header.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+/* eslint-disable linebreak-style */
```
Comment (mikenath223) [inline src/components/Header.js]: - [x] Kindly remove this eslint disable line and fix the issue in the code.

File: src/components/Navbar.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
+/* eslint-disable linebreak-style */
```
Comment (mikenath223) [inline src/components/Navbar.js]: - [x] Kindly remove this eslint disable line and fix the issue in the code.

File: src/components/QuoteData.js [hunk 5]
```diff
@@ -0,0 +1,77 @@
+/* eslint-disable linebreak-style */
```
Comment (mikenath223) [inline src/components/QuoteData.js]: - [x] Kindly remove this eslint disable line and fix the issue in the code.

File: src/index.js [hunk 6]
```diff
@@ -1,12 +1,16 @@
 /* eslint-disable linebreak-style */
```
Comment (mikenath223) [inline src/index.js]: - [x] Kindly remove this eslint disable line and fix the issue in the code.

File: src/pages/Home.js [hunk 7]
```diff
@@ -0,0 +1,46 @@
+/* eslint-disable linebreak-style */
```
Comment (mikenath223) [inline src/pages/Home.js]: - [x] Kindly remove this eslint disable line and fix the issue in the code.

File: src/pages/Mathematics.js [hunk 8]
```diff
@@ -0,0 +1,17 @@
+/* eslint-disable linebreak-style */
```
Comment (mikenath223) [inline src/pages/Mathematics.js]: - [x] Kindly remove this eslint disable line and fix the issue in the code.

File: src/pages/Quotes.js [hunk 9]
```diff
@@ -0,0 +1,27 @@
+/* eslint-disable linebreak-style */
```
Comment (mikenath223) [inline src/pages/Quotes.js]: - [x] Kindly remove this eslint disable line and fix the issue in the code.

File: README.md [hunk 10]
```diff
@@ -5,6 +5,10 @@ Math magicians is a website for all fans of mathematics. It is a Single Page App
 
 This project is part of the Microverse curriculum Module 3 Block 1. The main objective is about building your web application using React.
 
+![screenshot](img/pic1.png)
```
Comment (mikenath223) [inline README.md:8]: - [x] Please deploy the project online and include the live link in the readme.

File: src/App.js (hunk 1 above)
Comment (sja-thedude) [reply src/App.js]: Okay I fixed it

File: src/components/Footer.js (hunk 2 above)
Comment (sja-thedude) [reply src/components/Footer.js]: Okay I fixed it

File: README.md (hunk 10 above)
Comment (sja-thedude) [reply README.md:8]: Okay I deployed

**Review by ArthurGC — APPROVED** [review]
# Approved project!  💯 ✅ 🥳 🎈 🎉
Hi  @sja-thedude 

Amazing job implementing suggestions from the previous reviewer! 💯 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**